cognite_toolkit/_cdf_tk/utils/table_writers.py

```diff
@@ -18,8 +19,8 @@
     import pyarrow.parquet as pq
 
 FileFormat: TypeAlias = Literal["csv", "parquet", "yaml"]
-DataType: TypeAlias = Literal["string", "integer", "float", "boolean", "datetime", "date", "time", "json"]
-Rows: TypeAlias = list[dict[str, Any]]
+DataType: TypeAlias = Literal["string", "integer", "float", "boolean", "json"]
+Rows: TypeAlias = list[dict[str, str | int | float | bool | datetime | date | object | list | None]]
```

Comment:
Is a recursive definition of a json value more correct here?

Reply:
where `JsonVal: TypeAlias = None | str | int | float | bool | dict[str, "JsonVal"] | list["JsonVal"]`